data/update-lists.js

```diff
@@ -7,17 +10,22 @@ const args = process.argv.slice(2);
 
 if (args.length < 2) {
   console.error(
-    "Usage: node update-lists.js <Brave Services Key> <target version for brave list (i.e. 1.0.10268)>\n" +
-      "The component name is 'Brave Ad Block Updater'"
+    "Usage: BRAVE_SERVICE_KEY=<key> node update-lists.js <brave_list_version> <resource_list_version>\n" +
+      "The component names are 'Brave Ad Block Updater' and 'Brave Ad Block Resources Library'"
```

Comment:
Just a note that we updated the name of `Brave Ad Block Updater` to `Brave Default Adblock Filters` as of https://github.com/brave/adblock-resources/pull/268

Reply:
thanks, updated the name here: https://github.com/brave/adblock-rust/pull/500/commits/7912a9c36b54c772ab7882be8bf8966378745b57